src/components/navBar/navBar.js

```diff
@@ -175,6 +221,131 @@ function MdNavBarController($element, $scope, $timeout, $mdConstant) {
       deregisterTabWatch();
     }
   });
+
+    // Responsivity actions
+
+    /** @type {Object} */
```

Comment:
This file has explicit nullability for Objects and Functions. Please add them. If it can be null, the type should be ?Object. If it can't be null, the type should be !Object. (Ditto for functions)